addon/components/frost-select-dropdown.js

```diff
@@ -421,7 +424,12 @@ export default Component.extend({
         }
 
         if (filter) {
-          const pattern = new RegExp(filter.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'), 'gi')
+          let pattern
+          if (this.filterType === 'startsWith') {
```

Comment:
Why is this component being modified?  If it is necessary then this change requires the semver of this PR to be a MAJOR, right? since we're changing the behavior of this component?  Or has the default behavior remained in place and only new been added, in which case it can be a MINOR?

Reply:
The default behavior is still intact. I called the original behavior 'contains'

Reply:
Was it called `contains` before?  The questions is whether or not a user needs to make any changes to their existing code in order to obtain the current functionality.